.internal-ci/helm/fog-services/templates/supervisord-fog-ledger-configmap.yaml

```diff
@@ -10,15 +10,15 @@ data:
     [program:fog-ledger]
     priority=100
     environment=MC_SENTRY_DSN="%(ENV_FOG_LEDGER_SENTRY_DSN)s"
-    command=/usr/bin/ledger_server
+    command=/usr/bin/key_image_store
```

Comment:
This is an upstream question.  Why is this not called `fog-ledger-server` like `fog-view-server`

Should we change all the referencing names  to `key-image`?

Reply:
The Ledger service serves multiple APIs, one of which is the Key Image API. The Key Image API is the only one that has been converted to a router-type format. Hence why this is Key Image Store and not Ledger Store - it only backs the Key Image API.

Fog View on the other hand only serves one API. We could talk about `fog-view-server` vs `fog-view-store`, but that's the primary difference.